lib/fob/fragment_builder.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Fob.FragmentBuilder do
+  @moduledoc false
+  # Wyno cover? 
```

Comment:
since macros are expanded at compile time, :cover can't see function calls for 'em unless you write a test that does the compilation at test time, something like

```elixir
quote do
  build_from_existing(..)
end
|> Code.eval_quoted()
```

or similar

Reply:
I forgot I left that comment :) I'll add a test for coverage.